Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/yolo_impl.cpp

```diff
@@ -55,7 +67,7 @@ struct YoloModelImpl : public ModelInterface {
 		vector<cv::Mat> outs;
 
 	    static vector<cv::String> names;
-	    //Get output names from output layers
+	    // Get output names from output layers
```

Comment:
Does this matter?

Reply:
Yeah and this comment is from the tutorial provided online so I kept it here